package.json

```diff
@@ -5,5 +5,10 @@
     "start-frontend": "cd ./frontend && npm run start",
     "start-backend": "cd ./backend && npm run start",
     "start": "cd ./frontend && npm run start & cd ./backend && npm run start"
+  },
+  "dependencies": {
```

Comment:
@lnovitz 
Is there a reason to add dependencies to the top level package.json, instead of /frontend/package.json ?

Reply:
Mmm let me check